tests/container_test.py

```diff
@@ -10,6 +10,7 @@
 
 ENV_VAR = "ECHO_MESSAGE"
 ENV_VAR_VAL = "Hello World from docker compose!"
```

Comment:
Can someone tell me if/how `ENV_VAR` is used here?  I seem to recall some magic here, but I can't see it right now.

Reply:
It looks like that was lost in https://github.com/cisagov/skeleton-docker/commit/365fff926c7c11d266d4cc64148f2d2d9b283245 when testing was updated. Skimming over the tests I see some changes I'd like to make so I will PR to make those changes and restore the test that used that variable.

Reply:
I see - thanks for finding that.